dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_h2.sql

```diff
@@ -2185,3 +2185,64 @@ CREATE TABLE t_ds_listener_event
     KEY         idx_status (post_status),
     KEY         idx_event_sign (sign)
 );
+
+
+-- ----------------------------
+-- Table structure for jdbc registry
+-- ----------------------------
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_data`;
+CREATE TABLE `t_ds_jdbc_registry_data`
+(
+    `id`               bigint(11) NOT NULL AUTO_INCREMENT COMMENT 'primary key',
+    `data_key`         varchar(256) NOT NULL COMMENT 'key, like zookeeper node path',
+    `data_value`       text         NOT NULL COMMENT 'data, like zookeeper node value',
+    `data_type`        varchar(64)  NOT NULL COMMENT 'EPHEMERAL, PERSISTENT',
+    `client_id`        bigint       NOT NULL COMMENT 'client id',
+    `create_time`      timestamp    NOT NULL default current_timestamp COMMENT 'create time',
+    `last_update_time` timestamp    NOT NULL default current_timestamp COMMENT 'last update time',
+    PRIMARY KEY (`id`),
+    unique KEY `uk_t_ds_jdbc_registry_dataKey`(`data_key`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_lock`;
+CREATE TABLE `t_ds_jdbc_registry_lock`
+(
+    `id`          bigint(11) NOT NULL AUTO_INCREMENT COMMENT 'primary key',
+    `lock_key`    varchar(256) NOT NULL COMMENT 'lock path',
+    `lock_owner`  varchar(256) NOT NULL COMMENT 'the lock owner, ip_processId',
+    `client_id`   bigint       NOT NULL COMMENT 'client id',
+    `create_time` timestamp    NOT NULL default current_timestamp COMMENT 'create time',
+    PRIMARY KEY (`id`),
+    unique KEY `uk_t_ds_jdbc_registry_lockKey`(`lock_key`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_client_heartbeat`;
+CREATE TABLE `t_ds_jdbc_registry_client_heartbeat`
+(
+    `id`                  bigint(11) NOT NULL AUTO_INCREMENT COMMENT 'primary key',
+    `client_name`         varchar(256) NOT NULL COMMENT 'client name, ip_processId',
+    `last_heartbeat_time` bigint       NOT NULL COMMENT 'last heartbeat timestamp',
+    `connection_config`   text         NOT NULL COMMENT 'connection config',
+    `create_time`         timestamp    NOT NULL default current_timestamp COMMENT 'create time',
+    PRIMARY KEY (`id`)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+DROP TABLE IF EXISTS `t_ds_jdbc_registry_data_change_event`;
+CREATE TABLE `t_ds_jdbc_registry_data_change_event`
```

Comment:
is not data change like client online or offline.
```suggestion
CREATE TABLE `t_ds_jdbc_registry_client_change_event`
```

Reply:
This table is used to log the data change rather than client online or offline, client change is monitored by t_ds_jdbc_registry_client_heartbeat.

Reply:
get it.